client/utils/formatter.js

```diff
@@ -0,0 +1,11 @@
+export const capitalizeFirstLetterOfWords = (
```

Comment:
Is `capitalizeSentence` a valid name here? I didn't look too closely at what it does so I'm not 100% sure.

Reply:
it would be converting `hello_world` to `Hello World`. The function name of `capitalizeSentence` to me would mean converting to `Hello world`.

Maybe something like `capitalizeEachWord` is easier to process?

Reply:
Aaaaah, ok, up to you! I think both are good.